docs/DeveloperGuide.adoc

```diff
@@ -1850,6 +1898,34 @@ Use case ends.
 +
 Use case resumes at step 3.
 
+=== Statistic Report Window
+
+[discrete]
+==== Use case: UC23 - Displaying a statistic report
+
+*MSS*
+
+1. User requests to generate and display a statistic report.
+2. TA-Tracker retreives the list of completed sessions, and students.
+3. TA-Tracker processes the data retreived.
```

Comment:
I think you should combine these 2 so it looks more ping-pongy, also I feel like just here you should mention its the full list for all the modules or smth